tests/testthat/test-categories.R

```diff
@@ -34,6 +34,7 @@ with_mock_crunch({
 
     test_that("category slicers", {
         expect_true(is.categories(cats[1]))
+        expect_equal(cats[c("Female","Male")], cats[c(2,1)])
```

Comment:
Spaces.

Also, can you add some testing around the error handling (e.g. what happens if the strings you have don't match any category names)?